datafusion-proto/src/from_proto.rs

```diff
@@ -979,8 +1008,6 @@ impl TryFrom<&protobuf::LogicalExprNode> for Expr {
                     ScalarFunction::Sin => Ok(sin((&args[0]).try_into()?)),
                     ScalarFunction::Cos => Ok(cos((&args[0]).try_into()?)),
                     ScalarFunction::Tan => Ok(tan((&args[0]).try_into()?)),
-                    // ScalarFunction::Asin => Ok(asin(&args[0]).try_into()?)),
```

Comment:
I think the commented-out Asin, Acos, Concat, Array, Nullif, and md5 are removed?

Reply:
Nullif is not in `expr.rs`. will address in another PR.

Reply:
Opps,  forgot the deleted line, have added.